src/components/ProjectDisciplines.js

```diff
@@ -25,20 +28,59 @@ const mapStateToProps = (state) => ({
   user: state.user,
   isGuestUser: state.user.isGuestUser,
   isConnected: state.isConnected,
-  isFetching: state.isFetching
+  isFetching: state.isFetching,
+  pushPrompted: state.user.pushPrompted
 })
 
 const mapDispatchToProps = (dispatch) => ({
   setSelectedProjectTag(tag) {
     dispatch(setState('selectedProjectTag', tag))
   },
+  setPushPrompted(value) {
+    dispatch(setState('user.pushPrompted', value))
+    dispatch(syncUserStore())
+  },
 })
 
 export class ProjectDisciplines extends React.Component {
   constructor(props) {
     super(props);
   }
 
+  componentDidMount() {
+    if (this.shouldPromptForPermissions()) {
+      setTimeout(()=> {
+        this.promptRequestPermissions()
+      }, 500)
+    }
+  }
+
+  shouldPromptForPermissions() {
+    return ((Platform.OS === 'ios') && (!this.props.user.pushPrompted))
+  }
+
+  promptRequestPermissions = () => {
+    PushNotificationIOS.checkPermissions((permissions) => {
+      if (permissions.alert === 0){
+        AlertIOS.alert(
+          'Allow Notifications?',
+          'Zooniverse would like to occasionally send you info about new projects or projects needing help.',
+          [
+            {text: 'Not Now', onPress: () => this.requestIOSPermissions(false)},
+            {text: 'Sure!', onPress: () => this.requestIOSPermissions(true)},
```

Comment:
`Sure!` seems a bit awkward here. Can we make it `Ok`? Seems to be a bit more standard.